pages/getting-started/first-steps-with-docker.mdx

```diff
@@ -0,0 +1,491 @@
+```
+docker ps -a
+```
+
+Then start the container with: 
+```
+docker start <CONTAINER_ID> 
+```
+
+or 
+
+```
+docker start <CONTAINER_NAME> 
+```
+
+### Retrieve a Docker container ID
+
+Run the following command:
+
+```
+docker ps
+```
+
+You should get an output similar to this:
+
+```console
+CONTAINER ID   IMAGE                        COMMAND                  CREATED        STATUS        PORTS                                                                    NAMES
+45fa0f86f826   memgraph/memgraph-platform   "/bin/sh -c '/usr/bi…"   21 hours ago   Up 21 hours   0.0.0.0:3000->3000/tcp, 0.0.0.0:7444->7444/tcp, 0.0.0.0:7687->7687/tcp   admiring_almeida
+```
+
+You can shorten this ID to 4 letters if the ID remains unique, for example,
+`45fa`.
+
+### Retrieve a Docker container IP address
+
+To retrieve the Docker container IP address, first, you need [to retrieve its
+ID](#how-to-retrieve-a-docker-container-id) or use its [name](#container-name).
+
+Then run the following command if the container ID is `9397623cd87e`.
+
+```console
+docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' 9397623cd87e
+```
+
+### Browse files inside a Docker container
+
+To browse files inside a Docker container, first, you need [to retrieve its
+ID](#how-to-retrieve-a-docker-container-id) or use its [name](#container-name).
+
+Then run the following command if the container ID is `9397623cd87e`:
+
+```
+docker exec -it 9397623cd87e bash
+```
+
+To navigate through the container, use the following commands:
+
+- `ls` - list all the directories and files
+- `cd <directory>` - enter a directory
+- `cd ..` - leave the directory
+- `cat <file>` - list the content of a file
+
+You don't have to write file and directory names in full, once you write enough
+letters to have a unique string, press the `<TAB>` key.
+
+### Copy files from and to a Docker container
+
+To copy files from and to the Docker container, first, you need [to retrieve its
+ID](#how-to-retrieve-a-docker-container-id) or use its [name](#container-name).
+
+1. Place yourself in the local directory where you want to copy the file.
+
+2. Copy the file from the container to your current directory with the command:
+
+  ```
+  docker cp <CONTAINER ID>:<path_in_container> <local_file_name>
+  ```
+
+  Be sure to replace the `<CONTAINER ID>` parameter.
+
+  The example below will copy the configuration file to the user's Desktop:
+
+  ```
+  C:\Users\Vlasta\Desktop>docker cp bb3de2634afe:/etc/memgraph/memgraph.conf memgraph.conf
+  ```
+
+3. Copy the file from your current directory to the container with the command:
+
+  ```
+  docker cp  <local_file_name> <CONTAINER ID>:<path_in_container>
+  ```
+
+  Be sure to replace the `<CONTAINER ID>` parameter.
+
+  The example below will replace the configuration file with the one from the
+  user's Desktop:
+
+  ```
+  C:\Users\Vlasta\Desktop>docker cp memgraph.conf bb3de2634afe:/etc/memgraph/memgraph.conf
+  ```
+
+<Callout type="info">
```

Comment:
I don't like when whole chapters are info callouts... they are huge + why is this more valuable info than any other on this page?

Reply:
IMHO callouts have less vaulabe info that is relevant only for some special/edge cases :) I'll convert it to normal paragraph.